pyproject.toml

```diff
@@ -18,13 +18,13 @@ dependencies = [
     "apache-airflow >=2.0",
     "apache-airflow-providers-postgres",
     "apache-airflow-providers-snowflake",
+    "apache-airflow-providers-google",
     "pandas >=1.3.4",
     "s3fs",
     "snowflake-sqlalchemy ==1.2.0",
     "snowflake-connector-python[pandas]",
     "smart-open[all]>=5.2.1",
     "SQLAlchemy == 1.4.28",
```

Comment:
@ashb do we need to downgrade this?

Reply:
I think we need to downgrade it to Airflow's sqlAlchemy version - 1.3, because with latest pip we cannot have conflicting dependencies.